index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html lang="en">
+
+	<head>
+		<meta charset="utf-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+
+		<title>Anna Russett</title>
+	
+		<link rel="icon" type="image/x-icon" href="http://www. /favicon.ico">
+		<meta name="description" content="content">
+
+		<!-- for Google+ -->
```

Comment:
also related to that, here's[ a post ](https://moz.com/blog/meta-data-templates-123)that breaks these down a bit, but if it doesn't seem helpful there are lots of other blogs that breakdown these social meta tags in different ways